### PR TITLE
Bump the lestrrat-go/jwx for the sec fix

### PR DESCRIPTION
**Pull request by olegbespalov**
# What?

Updating the library to get the sec fix.

# Why?

https://github.com/grafana/k6/security/dependabot/29

### CODE REVIEW

**Review by joanlopez — APPROVED** [review]

**Review by oleiade — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -5,8 +5,8 @@ go 1.19
 require (
 	github.com/dop251/goja v0.0.0-20231027120936-b396bb4c349d
 	github.com/google/uuid v1.3.1
-	github.com/lestrrat-go/jwx/v2 v2.0.20
-	github.com/stretchr/testify v1.8.4
+	github.com/lestrrat-go/jwx/v2 v2.0.21
+	github.com/stretchr/testify v1.9.0
```
Comment (mstoykov) [inline go.mod:9]: Can we not bump this or any of the otehr ones :sweat_smile:

Comment (olegbespalov) [reply go.mod:9]: > Can we not bump this or any of the otehr ones 😅

I guess it's coming from the ` go get github.com/lestrrat-go/jwx/v2@v2.0.21` :cry:

Comment (olegbespalov) [reply go.mod:9]: https://github.com/lestrrat-go/jwx/blob/develop/v2/go.mod#L13

Comment (mstoykov) [reply go.mod:9]: Hmmm I guess you are right https://github.com/lestrrat-go/jwx/compare/v2.0.20...v2.0.21 :facepalm:

**Pull request merged**